packages/truffle-debugger/lib/data/selectors/index.js

```diff
@@ -279,6 +282,9 @@ const data = createSelectorTree({
         )
       ),
 
+      /**
+       * data.current.identifiers.native
+       */
```

Comment:
Thanks for adding the missing docstrings!

Reply:
No problem. :)